tests/smoke/test_notebooks_python.py

```diff
@@ -67,3 +67,22 @@ def test_fastai(notebooks):
     assert results["ndcg"] == pytest.approx(0.137494, TOL)
     assert results["precision"] == pytest.approx(0.124284, TOL)
     assert results["recall"] == pytest.approx(0.045587, TOL)
+
+@pytest.mark.smoke
+def test_adb_setup(notebooks):
+    notebook_path = notebooks["adb_setup"]
+    pm.execute_notebook(
+        notebook_path,
+        OUTPUT_NOTEBOOK,
+        kernel_name=KERNEL_NAME,
+        parameters=dict(path_to_recommenders_repo_root="./", 
+            record_for_tests=True, 
+            TOKEN="",
```

Comment:
This requires the TOKEN to be existing in the DevOps build machine. It is related to another pending issue #432 and hopefully we can have a more elegant way once there is building pipeline on Azure Databricks directly.

Reply:
yep. Open to all alternatives for testing that need secrets. Seems like an issue the group hasn't run into yet.